integration-test/src/test/resources/log4j2.properties

```diff
@@ -16,7 +16,7 @@ appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
 
 # Log files location
-property.logPath = ${sys:gravitino.log.path}
+property.logPath = ${sys:gravitino.log.path:-integration-test/build/integration-test.log}
```

Comment:
The behavior of `${sys:gravitino.log.path:-integration-test/build/integration-test.log}` is:
If `gravitino.log.path` is not set , the log file name is integration-test/build/integration-test.log. Otherwise the log path is the value of `gravitino.log.path`